scripts/installer-lang/vi.ps1

```diff
@@ -1,27 +1,27 @@
 [PSCustomObject]@{
     Welcome         = "
-╔═════════════════════════════════╗
-║ Chào Mừng tói SpotX cho Windows ║
-╚═════════════════════════════════╝"
+╔═════════════════════════════════════╗
+║ Chào mừng đến với SpotX cho Windows ║
+╚═════════════════════════════════════╝"
     Incorrect       = "Ối, sai giá trị rồi,"
     Incorrect2      = "hãy thử lại "
     Download        = "Lỗi khi tải xuống"
     Download2       = "Tải lại sau 5 giây..."
-    Download3       = "Tiếp tục lỗi"
-    Download4       = "Kiểm tra cài đặt mạng và chạy lại chương trình cài đặt"
+    Download3       = "Lại bị lỗi"
+    Download4       = "Hãy kiểm tra cài đặt mạng và chạy lại chương trình"
     Download5       = "Đang tải Spotify"
-    Download6       = "Lỗi curl"
+    Download6       = "Lỗi trong khi truyền tải dữ liệu"
```

Comment:
return the previous version of this line

Reply:
done